File: editorial_ch/chap2-2/prob2-2-4.cpp
#include <iostream>
using namespace std;

int N, A[109];
int Answer = 0;

int main() {
	// 輸入
	cin >> N;
	for (int i = 1; i <= N; i++) {
		cin >> A[i];
	}

	// 計算答案
	for (int i = 1; i <= N; i++) {
		Answer += A[i];
	}

	// 輸出
	cout << Answer % 100 << endl;
	return 0;
}

File: editorial_ch/chap2-2/prob2-2-4.java
import java.util.*;

class Main {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);

		// 輸入
		int N = sc.nextInt();
		int[] A = new int[N + 1];
		for (int i = 1; i <= N; i++) {
			A[i] = sc.nextInt();
		}

		// 計算答案
		int Answer = 0;
		for (int i = 1; i <= N; i++) {
			Answer += A[i];
		}

		// 輸出
		System.out.println(Answer % 100);
	}
};
